diff -up chromium-148.0.7778.96/build/config/compiler/BUILD.gn.omv~ chromium-148.0.7778.96/build/config/compiler/BUILD.gn
--- chromium-148.0.7778.96/build/config/compiler/BUILD.gn.omv~	2026-05-09 22:04:08.180964196 +0200
+++ chromium-148.0.7778.96/build/config/compiler/BUILD.gn	2026-05-09 22:04:29.395833141 +0200
@@ -1935,7 +1935,8 @@ config("sanitize_return") {
       # sanitizer) is enabled, they then do expensive debug like operations. We
       # want to suppress this behaviour since we want to keep performance costs
       # as low as possible while having these checks.
-      "-fsanitize-ignore-for-ubsan-feature=return",
+      # This flag doesn't exist in released clang.
+      #"-fsanitize-ignore-for-ubsan-feature=return",
     ]
   }
 }
